doc/ip/pinmux/pin_mappings.dot

```diff
@@ -485,8 +496,8 @@ digraph {
   gpio_rph:"gpio_0-27" -> rph:g4;
   gpio_rph:"gpio_0-27" -> rph:g5;
   gpio_rph:"gpio_0-27" -> rph:g6;
-  gpio_rph:"gpio_0-27" -> rph:g7_spi0_ce0;
-  gpio_rph:"gpio_0-27" -> rph:g8_spi0_ce1;
+  gpio_rph:"gpio_0-27" -> rph:g7_spi0_ce1;
+  gpio_rph:"gpio_0-27" -> rph:g8_spi0_ce0;
```

Comment:
Would you be able to remove the SPI suffixes from the Raspberry Pi Header pins?

```suggestion
  gpio_rph:"gpio_0-27" -> rph:g7;
  gpio_rph:"gpio_0-27" -> rph:g8;
```

Reply:
Added in the latest commit.